src/js/providers/flash.js

```diff
@@ -222,7 +222,6 @@ define([
                         this.sendEvent(events.JWPLAYER_MEDIA_ERROR, {
                             message: 'Error loading media: File could not be played'
                         });
-                        this.setState(states.IDLE);
```

Comment:
this.setState(states.IDLE); was removed because setting that state is handled in the controller.js with the JWPLAYER_MEDIA_ERROR handler on line 98.  That changes the state so we don't have to put that in every provider.   As far as the public API we're in about the same boat as we are with the complete state.  jwplayer().getState() returns 'error' in the error state as the jwplayer().getState() returns 'complete' in the complete state.  To match the api from jw6 we'd probably have to have both of them return 'idle' in their states.